x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/metrics/metrics_grid.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiFlexGrid, EuiFlexItem, EuiFlexGroup, EuiText, EuiI18n } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { MetricChart } from './metric_chart';
+
+const DEFAULT_BREAKDOWN_SIZE = 20;
+
+export const MetricsGrid = () => {
+  return (
+    <>
```

Comment:
**nit:** you can remove this fragment as the component returns a single `EuiFlexGroup` node